### ISSUE TITLE
Serving | Op type not registered 'PaddedBatchDatasetV2' in binary 

### ISSUE THREAD

**Issue by nadavb** [tensorflow/tensor2tensor#1166]
After upgrading the t2t, and successfully training a model on a GPU machine, I tried taking the trained model, export it on a CPU machine, and serve it using tensorflow_model_server
When I run the tensorflow_model_server exec, I get: "Op type not registered 'PaddedBatchDatasetV2' in binary "

I made sure the tensorflow_model_server is up to date using "sudo apt-get upgrade tensorflow-model-server" - but it didn't help

### Environment information
```
OS: ubuntu 16

$ pip freeze | grep tensor
tensorboard==1.11.0
tensorflow==1.11.0
tensorflow-hub==0.1.1
tensorflow-serving-api==1.11.1
tensorflow-tensorboard==1.5.0

$ python -V
Python 2.7.12
```

**Comment by mehmedes** [comment]
Is your model based on a ```_packed``` problem?

**Comment by nadavb** [comment]
@mehmedes I don't know what is a _packed problem. I am using the standard translate_ende problem, just with my own dataset. Is this a _packed problem?

**Comment by mehmedes** [comment]
Packing does:
https://github.com/tensorflow/tensor2tensor/blob/a4f958a887f4f4466644dd0602bdd33985d61dd7/tensor2tensor/data_generators/text_problems.py#L166
I remember getting the "Op type not registered 'PaddedBatchDatasetV2" when using ```PROBLEM=translate_ende_wmt32k_packed``` during data generation and running the server model as ```PROBLEM=translate_ende_wmt32k_packed```. 
Exporting it and running it as ```PROBLEM=translate_ende_wmt32k``` instead of ```PROBLEM=translate_ende_wmt32k_packed``` helped me that time.

**Comment by nadavb** [comment]
I never used the _packed keyword, I simply export with:

  --model=transformer \
  --hparams_set=transformer_base \
  --problem=translate_ende_wmt32k

What am I missing?

**Comment by nadavb** [comment]
Okay, problem solved.
I had to upgrade from tensorflow_model_server 1.11. The thing is that it was tricky to upgrade, because apt-get upgrade tensorflow-model-server didn't work by itself - it didn't print any error message regarding the fact that I don't have the latest version. I had to remove the package completely, to do apt-get update and only then to install it again. Then everything worked like a charm.
Thanks!
